libs/brackets-server/embedded-ext/brackets-minify/node/minifyjs.js

```diff
@@ -0,0 +1,92 @@
+(function() {
+    "use strict";
+
+    var path     = require('path'),
+        fs       = require('fs'),
+        mkpath   = require('mkpath'),
+        UglifyJS = require("uglify-js");
+
```

Comment:
single qutation and double quatation mark seems to be mixed on same file.
coding style ?

Reply:
Fixed